### ISSUE TITLE
Tests fail with babel v7

### ISSUE THREAD

**Issue by tleunen** [tleunen/eslint-import-resolver-babel-module#74]
It seems it was working fine with babel 7 beta.2, but the new versions fail.

cc @fatfisz

**Comment by fatfisz** [comment]
Will check in a bit!

**Comment by fatfisz** [comment]
Heads up: I'm getting weird errors when trying to run the tests:
```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for babel-core@7.0.0-beta
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

This is the case for all 7.0.0-beta packages. I'm trying to get around this right now.

**Comment by tleunen** [comment]
Hmm.. Yep they changed it to be `@babel/core` I believe since beta.X. But even after changing them, I still got the error. I'll have a bit more time tonight to investigate further too

**Comment by fatfisz** [comment]
For now I just set versions to 7.0.0-beta.3 and it works, but I couldn't find a solution. Btw. I checked with yarn and it just asked me to select from the list of versions, so maybe the resolution only works with we're dealing with X.Y.Z?

**Comment by fatfisz** [comment]
I got something to work on v.7.0.0-beta.3, will try other versions next. I probably won't be able to do much today, so look forward to tomorrow ;)

**Comment by tleunen** [comment]
Fixed in #78
